src/test/java/tr/com/srdc/cda2fhir/testutil/generator/CDAMedicationSectionComponentGenerator.java

```diff
@@ -0,0 +1,52 @@
+package tr.com.srdc.cda2fhir.testutil.generator;
+
+import java.util.List;
+
+import org.openhealthtools.mdht.uml.cda.Component3;
+import org.openhealthtools.mdht.uml.cda.SubstanceAdministration;
+import org.openhealthtools.mdht.uml.cda.consol.MedicationsSection;
+import org.openhealthtools.mdht.uml.hl7.datatypes.CE;
+import org.openhealthtools.mdht.uml.hl7.datatypes.II;
+import org.openhealthtools.mdht.uml.hl7.vocab.ActClass;
+import org.openhealthtools.mdht.uml.hl7.vocab.ActMood;
+import org.openhealthtools.mdht.uml.hl7.vocab.ActRelationshipHasComponent;
+
+import tr.com.srdc.cda2fhir.testutil.CDAFactories;
+
+public class CDAMedicationSectionComponentGenerator {
+
+	private List<SubstanceAdministration> substanceAdministrations;
+	static final public String DEFAULT_TEMPLATE_ID = "2.16.840.1.113883.10.20.22.2.1";
```

Comment:
Prefer generators as fields instead of actual data.

Reply:
Gotchya, I guess I thought that the template ID matching the correct ID for the actual Resource would be important for the CCDTransformer recognizing them but I don't know if that's true.